.github/workflows/tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Tests
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: ["2.7", "3.8", "3.9"]
+        os: [ubuntu-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup python
+      uses: actions/setup-python@v2
+
+    - name: Python ${{ matrix.python-version }}
+      run: |
+        pip install coveralls
+        pip install -e . --force-reinstall
+
+    - name: Tests
+      run: |
+        coverage run setup.py test
+
+    # - name: Coveralls
+    #   uses: coverallsapp/github-action@master
+    #   with:
+    #     github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I left this one commented out b/c someone with admin privileges will need to set that token up. (I can remove this part of the code if you don't want it there.)